superset-frontend/src/components/ButtonGroup/index.tsx

```diff
@@ -30,22 +31,28 @@ export default function ButtonGroup(props: ButtonGroupProps) {
       role="group"
       className={className}
       css={{
-        '& :nth-of-type(1):not(:nth-last-of-type(1))': {
+        display: 'flex',
+        '& > :nth-of-type(1):not(:nth-last-of-type(1))': {
```

Comment:
Now that we have icons in buttons sometimes (which was always supported by AntD), I made these styles more specific, so they apply to the buttons themselves, and not elements _within_ the buttons.